Fix glossary (#2054)
aivivn/d2l-vn: glossary.md

| batch size                             | kích thước batch               | [https://git.io/JvXdK](https://git.io/JvXdK) |
| beam search                            | tìm kiếm chùm                  | [https://git.io/Jf9Nl](https://git.io/Jf9Nl) |
| benchmark                              | đánh giá xếp hạng              | [https://git.io/JvQxY](https://git.io/JvQxY) |
| bias (model parameter)                 | hệ số điều chỉnh               | [https://git.io/Jvopx](https://git.io/Jvopx) |
| bias (of an estimator)                 | độ chệch                       | [https://git.io/JvQxO](https://git.io/JvQxO) |
| bias-variance tradeoff                 | đánh đổi độ chệch - phương sai | [https://git.io/JvQA6](https://git.io/JvQA6) |
| bidirectional recurrent neural network | mạng nơ-ron hồi tiếp hai chiều | [https://git.io/JJeal](https://git.io/JJeal) |
| big data                               | big data                       |                                              |
| binomial distribution                  | phân phối nhị thức             | [https://git.io/JvohQ](https://git.io/JvohQ) |

> | blackbox dev set                       | tập phát triển Blackbox        | [https://git.io/JvQx3](https://git.io/JvQx3) |

Reviewer: ```suggestion
| blackbox dev set                       | tập phát triển blackbox        | [https://git.io/JvQx3](https://git.io/JvQx3) |
```

| linear discriminant analysis (LDA) | phân tích biệt thức tuyến tính | [https://git.io/Jvojw](https://git.io/Jvojw) |
| linear form                        | dạng tuyến tính                | [https://git.io/JvohV](https://git.io/JvohV) |
| linear independence                | độc lâp tuyến tính             | [https://git.io/JvKet](https://git.io/JvKet) |
| linear programming                 | quy hoạch tuyến tính           | [https://git.io/JvohV](https://git.io/JvohV) |
| linear regression                  | hồi quy tuyến tính             |                                              |
| local maximum                      | cực đại                        |                                              |
| local minimum                      | cực tiểu                       |                                              |
| locality                           | tính cục bộ                    |                                              |
| log-likelihood function            | hàm log hợp lý                 | [https://git.io/Jvopx](https://git.io/Jvopx) |
| logistic regression                | hồi quy logistic               |                                              |

> | logit (trong softmax)              | logit                          | [https://git.io/JvohR](https://git.io/JvohR) |

Reviewer: mình không sao lại đóng mở ngoặc softmax ở đây nhỉ

> | logit (trong softmax)              | logit                          | [https://git.io/JvohR](https://git.io/JvohR) |

Reviewer: ```suggestion
| logit (in softmax)              | logit                          | [https://git.io/JvohR](https://git.io/JvohR) |
```

> | logit (trong softmax)              | logit                          | [https://git.io/JvohR](https://git.io/JvohR) |

Reviewer: @minhduc0711

> | logit (trong softmax)              | logit                          | [https://git.io/JvohR](https://git.io/JvohR) |

Reviewer 2: sót sao ông giáo?

> | logit (trong softmax)              | logit                          | [https://git.io/JvohR](https://git.io/JvohR) |

Reviewer: sao k sửa từ `trong` nhỉ. mình tưởng thống nhất cột trái là tiếng Anh?

> | logit (trong softmax)              | logit                          | [https://git.io/JvohR](https://git.io/JvohR) |

Reviewer 2: thì đúng rồi, đã update rồi đấy ông giáo.

## P
| English                            | Tiếng Việt                   | Thảo luận tại                                |
|------------------------------------|------------------------------|----------------------------------------------|
| p-value                            | trị số p                     | [https://git.io/Jvoja](https://git.io/Jvoja) |
| padding                            | đệm                          | [https://git.io/Jfe1I](https://git.io/Jfe1I) |
| partition function                 | hàm phân hoạch               | [https://git.io/JvQxE](https://git.io/JvQxE) |
| pattern recognition                | nhận dạng mẫu                | [https://git.io/JvKeL](https://git.io/JvKeL) |
| penalty                            | lượng phạt                   | [https://git.io/JvQAP](https://git.io/JvQAP) |
| perceptron                         | perceptron                   | [https://git.io/JvohC](https://git.io/JvohC) |
| performance                        | chất lượng                   | [https://git.io/JvQx4](https://git.io/JvQx4) |

> | perplexity (in information theory) | perplexity                   | [https://git.io/Jf9KY](https://git.io/Jf9KY) |

Reviewer 3: ```suggestion
| perplexity (metric NLP) | perplexity                   | [https://git.io/Jf9KY](https://git.io/Jf9KY) |
```
Chỗ này hình như không đúng lắm. perplexity là một metric được cộng đồng NLP tạo ra dựa trên entropy thôi chứ không phải gốc từ information theory.

Author: oh vậy à, my bad

Reviewer: ```suggestion
| perplexity (a metric in NLP) | perplexity                   | [https://git.io/Jf9KY](https://git.io/Jf9KY) |
```
Chỗ này hình như không đúng lắm. perplexity là một metric được cộng đồng NLP tạo ra dựa trên entropy thôi chứ không phải gốc từ information theory.
